src/m365/entra/commands/user/user-get.spec.ts

```diff
@@ -222,95 +227,7 @@ describe(commands.USER_GET, () => {
     await assert.rejects(command.action(logger, { options: { id: '68be84bf-a585-4776-80b3-30aa5207aa22' } } as any),
       new CommandError(`Resource '68be84bf-a585-4776-80b3-30aa5207aa22' does not exist or one of its queried reference-property objects are not present.`));
   });
-
-  it('fails to get user when user with provided id does not exists', async () => {
```

Comment:
Is there a reason why we're removing these test cases? They seem like valid issues that we should test

Reply:
Now, I'm using `aadUser` util in the command and stub for `aadUser` in tests and these removed cases are covered in `aadUser.spec.ts`. In fact, only the case when no user is found, because you can't get two or more users with the same email or UPN.

Reply:
> Now, I'm using `aadUser` util in the command and stub for `aadUser` in tests and these removed cases are covered in `aadUser.spec.ts`. In fact, only the case when no user is found, because you can't get two or more users with the same email or UPN.

That's however an internal implementation detail. The idea of tests is, that at least theoretically, the test cases should stand even when we change the implementation. If it's theoretically possible that a particular test case could happen, then let's ensure that we test for it and can verify that our code can handle it.

Reply:
The test `fails to get user when user with provided id does not exists` is now covered by another test `correctly handles user not found`.

The tests `handles error when multiple users with the specified email found` and `handles selecting single result when multiple users with the specified email found and cli is set to prompt` are use cases that can never happen. You can't have two users with the same email address or user principal name in Entra.

The tests `fails to get user when user with provided user name does not exists` and `fails to get user when user with provided email does not exists` are still valid and will add them.